third_party/amd/backend/compiler.py

```diff
@@ -169,6 +169,9 @@ def make_llir(src, metadata, options):
         pm = ir.pass_manager(mod.context)
         pm.enable_debug()
         amd.passes.ttgpuir.add_decompose_unsupported_conversions(pm, options.arch)
+        # experimental parameter, specifies custom LDS usage limit
```

Comment:
Can you elaborate on this parameter? What does it mean when it's set to a non-zero and zero?
Especially when it's set to non-zero value, does it mean the total LDS usage is guaranteed to be lower than that? Or is it just a hint?